dev/manual_tests/test/mock_image_http.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:async';
+import 'package:flutter/foundation.dart' show ValueGetter;
```

Comment:
No need to use "show" with our own packages (here and above).
It's fine to do so if you want to, so you're welcome to leave it like this. But in general it's not necessary.

Reply:
(We do it specifically for dart:ui because we want to discourage depending on dart:ui.)

Reply:
I actually deliberately did it here, after looking at some other file doing something similar and wondering what to import for ValueGetter... :)